gunicorn.conf.py

```diff
@@ -1 +1,3 @@
-import os
+reload = True
```

Comment:
```suggestion
# https://docs.gunicorn.org/en/stable/configure.html#configuration-file
reload = True
```

The docs seem to suggest this could be called anything: "(e.g. gunicorn.conf.py)" is this actually just the only value it looks for or do we set this somewhere else?

About the paster stuff (I think this is how we got here in the first place), but the docs say:

> Any parameters that Gunicorn knows about will automatically be inserted into the base configuration. Remember that these will be overridden by the config file and/or the command line.

I can't remember where we got with this, but is this just a big fat lie, even with the `--paste` argument?

Reply:
I think "it could be anything" means you can pass `-c anything.py` as a command line argument. By default it looks for `gunicorn.conf.py` in the CWD: https://docs.gunicorn.org/en/latest/configure.html?highlight=gunicorn.conf.py#configuration-overview

Reply:
> About the paster stuff (I think this is how we got here in the first place), but the docs say:
> 
> > Any parameters that Gunicorn knows about will automatically be inserted into the base configuration. Remember that these will be overridden by the config file and/or the command line.
> 
> I can't remember where we got with this, but is this just a big fat lie, even with the --paste argument?

Honestly, I don't know